TileFormats/FeatureTable/schema/i3dm.featureTable.schema.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema" : "http://json-schema.org/draft-04/schema#",
+  "id" : "i3dm.featureTable.schema.json",
+  "title" : "Instanced 3D Model Feature Table",
+  "type" : "object",
+  "description": "A set of Instanced 3D Model semantics that containper-feature values defining the position and appearance properties for instanced models in a tile.",
+  "allOf" : [
+    {
+      "$ref" : "featureTable.schema.json"
+    },
+    {
+      "properties": {
```

Comment:
Each of these need to define the value values, e.g., is it a scalar or an array, min/max, etc.